arangod/Transaction/Methods.cpp

```diff
@@ -1164,14 +1175,17 @@ Future<OperationResult> transaction::Methods::insertLocal(std::string const& cna
           prevDocResult.revisionId().isSet()) {
         TRI_ASSERT(didReplace);
 
+        TRI_ASSERT(res.ok());
```

Comment:
That assertion is only executed if `res` is `TRI_ERROR_ARANGO_CONFLICT` and thus _not_ ok. Wasn't that triggered by jenkins?

Reply:
Good catch. This is a leftover. Will remove the wrong assert!

Reply:
However, this shows that this code segment has close to zero test coverage. 😟